packages/syria/README.md

```diff
@@ -0,0 +1,38 @@
+# SY Syria
+
+Contains all the regions of Syria:
+* Al Hasakah
+* Al Ladhiqiyah
+* Al Qunaytirah
+* Ar Raqqah
+* As Suwayda
+* Dayr az Zawr
+* Halab
+* Hamah
+* Hims
+* Idlib
+* Rif Dimashq
+* Tartus
```

Comment:
Could you use the English names please? If you know an official or better source than wikipedia for the English names, feel free to share it

Reply:
Yes , i used the English names as provided by the wiki, the pronunciation is way off, but they seem to be the standard globaly